app/controllers/claims/schools/claims_controller.rb

```diff
@@ -1,3 +1,51 @@
 class Claims::Schools::ClaimsController < ApplicationController
   include Claims::BelongsToSchool
+
+  def index
+    render locals: { claims: @school.claims }
```

Comment:
Is there a strong reason behind passing this through as a local instead of an instance variable?

```suggestion
    @claims = @school.claims
```

Reply:
Personally, I prefer to use locals over instance variables. For this index method instance, variables work but when you have to render partials, like in the edit controller method and view I think you need to use locals. So because of this I prefer to always use locals. But I know there are places where we use instance variables and places where we use both 🤷🏻

Reply:
As far as I can tell, the distinction of using instance variables vs locals is:

- Instance Variables for passing variables from Controllers down to Views
- Locals when passing variables from a View into a Partial

Would you object to this?

I'm a personal fan of [this gem](https://github.com/hashrocket/decent_exposure) that cleans this up a lot, but it's not heavily maintained. But this is just to share "exposure" 😆

Reply:
I don't object to the distinction, I'm just saying that we can just use locals to do both to reduce a little context switching. 

About the gem, it looks good but I feel like we are bringing a 3rd party dependency to do something we can easily do ourselves. and if we do decide to use it should we ask placements as well?

Reply:
For now I can just change the methods that don't require locals to use instance variables

Reply:
Addressed in https://github.com/DFE-Digital/itt-mentor-services/pull/140/commits/319fcb453b11564aaa3b3be6843e8138378a34e4

Reply:
Sorry, I don't think I was clear in my latest comment. I meant that there's a nice gem out there, but I don't trust it enough to add into this project.